src/nextjournal/clerk/analyzer.clj

```diff
@@ -544,12 +549,21 @@
 #_(dep/immediate-dependencies (:graph (build-graph "src/nextjournal/clerk/analyzer.clj"))  #'nextjournal.clerk.analyzer/long-thing)
 #_(dep/transitive-dependencies (:graph (build-graph "src/nextjournal/clerk/analyzer.clj"))  #'nextjournal.clerk.analyzer/long-thing)
 
+(defn strip-form-meta [form]
+  (clojure.walk/postwalk
+   (fn [v]
+     (if (or (instance? clojure.lang.IObj v)
+             (instance? clojure.lang.IMeta v))
```

Comment:
Hi @craig-latacora thank you for your pr!

If I can see correctly, a more correct check would be just `(instance? clojure.lang.IObj v)`. While it's unlikely that this `form` contains complex non-literal values, strictly looking at the interface definitions an object needs to implement `IObj` to be called `vary-meta` upon. For instance a var is IMeta but not IObj.

Reply:
I've pushed your changes + some cleanup internally to https://github.com/nextjournal/clerk/compare/latacora/strip-meta?expand=1.